content/tracing/setup/docker.md

```diff
@@ -46,35 +46,85 @@ As with DogStatsD, traces can be submitted to the agent from other containers ei
 ### Using Docker links
 
 ```
-docker run  --name my_container           \
-            --all_your_flags              \
-            --link dd-agent:dd-agent    \
+docker run  --name my_container \
+            --all_your_flags \
+            --link dd-agent:dd-agent \
             my_image
 ```
 
-It expose `DD_AGENT_PORT_8126_TCP_ADDR` and `DD_AGENT_PORT_8126_TCP_PORT` as environment variables. Your application tracer can be configured to submit to this address.
+It expose `DD_AGENT_PORT_8126_TCP_ADDR` and `DD_AGENT_PORT_8126_TCP_PORT` as environment variables. Your application tracer must be configured to submit traces to this address.
 
-An example in Python:
+
+#### Python
 
 ```python
 import os
 from ddtrace import tracer
+
 tracer.configure(
-    hostname=os.environ["DD_AGENT_PORT_8126_TCP_ADDR"],
-    port=os.environ["DD_AGENT_PORT_8126_TCP_PORT"]
+    hostname=os.environ['DD_AGENT_PORT_8126_TCP_ADDR'],
+    port=os.environ['DD_AGENT_PORT_8126_TCP_PORT'],
+)
+```
+
+#### Ruby
+
+```ruby
+Datadog.configure do |c|
+  c.tracer hostname: ENV['DD_AGENT_PORT_8126_TCP_ADDR'],
+           port: ENV['DD_AGENT_PORT_8126_TCP_PORT']
+end
+```
+
+#### Go
+
+```go
+import (
+    "os"
+
+    ddtrace "github.com/DataDog/dd-trace-go/opentracing"
 )
+
+func main() {
+    config := ddtrace.NewConfiguration()
+    config.AgentHostname = os.GetEnv("DD_AGENT_PORT_8126_TCP_ADDR")
+    config.AgentPort = os.GetEnv("DD_AGENT_PORT_8126_TCP_PORT")
+
+    tracer, _, _ := ddtrace.NewTracer(config)
+}
+```
+
+#### Java
```

Comment:
@realark can you double check this section?

Reply:
:+1: Looks good!